Add a readme file

hanselomondi opened the pull request:
### Changes being made
Adding a readme file.

### Context of the changes
This is the initial version of the readme file.

### Have you followed the collaboration guidelines?
- [x] I pulled all possible changes from upstream before adding my changes
- [x] My files passed all betty styling and documentation guidelines
- [x] I've tested my program/source code and it works as intended
- [x] Proper commit messages

### Other context
None.

Ogoa commented on README.md:
```diff
@@ -3,19 +3,29 @@ In this project, we implement a simple shell that functions in a similar way to
+
+
+
 `getline.c`<br>
```
A comment

Ogoa commented on README.md:
````diff
@@ -3,19 +3,29 @@ In this project, we implement a simple shell that functions in a similar way to
 `_getline` reads an entire line of text from an input stream. On success, it returns the number of characters read, otherwise it returns `-1` on failure or upon encountering the `EOF`.<br>
 The function prototype is:
 ```
-int _getline(const char **lineptr, int *n, FILE *stream);
````
A small demonstration

hanselomondi committed:
Add a readme file

This is the initial version of the readme file.

hanselomondi committed:
Add custom string-printing functions

hanselomondi committed:
Add handling of empty spaces passed as a command

Previously, the shell was not able to handle the case where the command
entered was one or more empty spaces. This was changed to make the loop
jump back to the top in the case where an empty space was passed as the
command.
